include/envoy/http/access_log.h

```diff
@@ -114,6 +114,16 @@ class RequestInfo {
    * Set whether the request is a health check request or not.
    */
   virtual void healthCheck(bool is_hc) PURE;
+
+  /**
+   * Set the downstream address
+   */
+  virtual void setDownstreamAddress(const std::string& address) PURE;
```

Comment:
Please remove this from the interface per the previous discussion. It doesn't make sense to be able to set the downstream address on the request info at the interface level IMO.